qiskit/wrapper/configrc/_settings.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2018, IBM.
+#
+# This source code is licensed under the Apache License, Version 2.0 found in
+# the LICENSE.txt file in the root directory of this source tree.
+
+"""Module that holds the runtime location of qiskitrc file.
+"""
+
+QISKIT_RC_FILE = None
+
+REGISTER_CALLED = 0
```

Comment:
Would it be possible to make better use of `qiskit-core/qiskit/wrapper/defaultqiskitprovider.py` for keeping track of the registered backends and their names, instead of a global variable? Currently it [uses a list](https://github.com/QISKit/qiskit-core/blob/master/qiskit/wrapper/defaultqiskitprovider.py#L26) of providers, but by extending it to a dict it seems it would simplify the handling of the providers names and split the functionality better - with the big benefit of having a single instance where that information is stored instead of global scope.

Reply:
The `_settings` does not keep track of the registered backends, it only is used for storing runtime variables.  The backends are stored as usual, using the new `Provider.name` functionality to understand which are which.